praw/models/reddit/mixins/inboxtoggleable.py

```diff
@@ -29,6 +33,10 @@ def disable_inbox_replies(self):
 
     def enable_inbox_replies(self):
         """Enable inbox replies for the item.
+        
+        .. note::
+        
+            This can only apply to items created by the authenticated user.
```

Comment:
```suggestion

        .. note::

            This can only apply to items created by the authenticated user.
```
Look like there is trailing whitespace that is causing issues.